microsoft/testsuites/performance/common.py

```diff
@@ -339,10 +341,15 @@ def perf_ntttcp(  # noqa: C901
                 client_nic_name if client_nic_name else client.nics.default_nic
             )
             dev_differentiator = "Hypervisor callback interrupts"
+
+        server_ip_address = server.internal_address
+
         server_lagscope.run_as_server_async(
-            ip=lagscope_server_ip
-            if lagscope_server_ip is not None
-            else server.internal_address
+            ip=(
```

Comment:
When would `lagscope_server_ip` be empty?

Reply:
That is a good question.

The definition of perf_ntttcp has parameter " lagscope_server_ip: Optional[str] = None,"